### PR TITLE
apprenticeship post

### PR DESCRIPTION
**Pull request by littleredninja**
#### Who should review this PR?

@ameliapadua @SuzIllini2 @johnle192 
#### Anything the reviewer(s) should know to review effectively?

:heart_eyes_cat: I appreciate you! :heart_eyes_cat:

### CODE REVIEW

File: _posts/2015-08-31-developing-a-personal-project.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Ultimately, I picked a project that I was excited about and one that involved technologies I was interested in learning. I tried to scope the project so that it wasn't something that would be urgently needed for the business so that I could take my time learning and experimenting, but strived for something that could be useful and had room to grow or could be build upon. I spent some time talking through some strategies with my mentor, such as where the project should live and what tools I should use to build it. Eventually, I set about trying to make it happen.
```
Comment (pdadhaniya) [inline _posts/2015-08-31-developing-a-personal-project.md]: In the 2nd sentence, `built` instead of `build`

**Comment by pdadhaniya** [comment]
:+1:

File: _posts/2015-08-31-developing-a-personal-project.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: Developing a Personal Project
+date: 2015-08-31 10:45
+comments: true
+author: Jennie Chamberlin
+---
+
+Being an apprentice at Trunk Club has been a great experience, and part of the experience is developing your own personal project. Over the course of my apprenticeship, I had to find an idea, pitch a product, and learn about the associated technologies and systems required to make the project come to life. The whole process taught me a lot about what it means to be a developer with an entrepreneurial spirit.
```
Comment (jeffmeyers) [inline _posts/2015-08-31-developing-a-personal-project.md]: Tense switch `has` -> `is`

**Comment by jeffmeyers** [comment]
:+1: outside of comments

**Pull request merged**